milestone2.ts

```diff
@@ -83,34 +83,48 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage: GetNumber = (data) =>{
+  let sum = 0;
+  for(let i = 0; i < data.length; i++)
+  {
+    sum+= data[i].age;
+  }
+  return sum/data.length
+}; // Define lambda function here
 
-//console.log(findAverage(userData));
+console.log(findAverage(userData));
 
 // Question 2: Data Handling
 /* Use data handling function(s) to find the first index of someone from San Francisco (SF).
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data)=>{
+  return data.findIndex(x =>x.city === "SF");
+}; // Code here
 
-//console.log(findIndexAns(userData));
+console.log(findIndexAns(userData));
 
 // Question 3: Filtering data
 /* Use data handling function(s) to find all of the people from California (CA) over an age threshold n
  */
 type GetVolunteers = (data: Volunteer[], minAge: number) => Volunteer[];
-let findCAOverN: GetVolunteers; // Code here
+let findCAOverN: GetVolunteers = (data, n) =>{
+  let filter1 = data.filter(x =>x.state === "CA");
+  return filter1.filter(x =>x.age >n);
```

Comment:
This can be shortened with a logical and (&&)
```suggestion
  let filter1 = data.filter(x =>x.state === "CA" && x.age >n);
  return filter1
```